packages/react/src/CTAForm/CTAForm.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {CTAForm} from './CTAForm'
+import {FormControl, TextInput, Checkbox, Text, InlineLink} from '../index'
+
+export default {
+  title: 'Components/CTAForm',
+  component: CTAForm
+} as ComponentMeta<typeof CTAForm>
+
+export const Playground: ComponentStory<typeof CTAForm> = _args => (
+  <CTAForm>
+    <CTAForm.Input>
+      <FormControl required>
+        <FormControl.Label>Your Work Email Address</FormControl.Label>
+        <TextInput placeholder="name" />
+      </FormControl>
+    </CTAForm.Input>
+    <CTAForm.Confirm>
+      <FormControl required>
+        <FormControl.Label>
+          <Text size="300" variant="muted">
+            I agree to the <InlineLink href="www.github.com">Privacy Policy</InlineLink> and{' '}
```

Comment:
Will 404 for anyone that tries to click this currently.

```suggestion
            I agree to the <InlineLink href="https://github.com">Privacy Policy</InlineLink> and{' '}
```